docs/markdown/Pkgconfig-module.md

```diff
@@ -25,14 +28,13 @@ keyword arguments.
 - `libraries` a list of built libraries (usually results of
   shared_library) that the user needs to link against. Arbitrary strings can
   also be provided and they will be added into the `Libs` field. Since 0.45.0
-  dependencies of built libraries will be automatically added to `Libs.private`
-  field. If a dependency is provided by pkg-config then it will be added in
-  `Requires.private` instead. Other type of dependency objects can also be passed
-  and will result in their `link_args` and `compile_args` to be added to `Libs`
-  and `Cflags` fields.
+  dependencies of built libraries will be automatically added, see the
+  [Implicit dependencies](#Implicit_dependencies) section below for the exact
+  rules.
 - `libraries_private` list of built libraries or strings to put in the
-  `Libs.private` field. Since 0.45.0 it can also contain dependency objects,
-  their `link_args` will be added to `Libs.private`.
+  `Libs.private` field. Since 0.45.0 dependencies of built libraries will be
```

Comment:
I think this actually happened in 0.46 - if that's the case, what about fixing it as well while this line is being changed?

https://github.com/mesonbuild/meson/blob/master/docs/markdown/Release-notes-for-0.46.0.md#improved-generation-of-pkg-config-files-for-static-only-libraries